### ISSUE TITLE
No Start Menu icon in Linux Mint

### ISSUE THREAD

**Issue by SportingChap** [dangeredwolf/ModernDeck#331]
Running ModernDeck in Linux Mint 20.3, Cinnamon 5.2.7, there is no start menu icon:

![image](https://user-images.githubusercontent.com/67240565/178113120-71aaee1d-e504-41ac-9aa2-7642d3b7fa9b.png)

(Could be related to #330 and #323?)

**Comment by dangeredwolf** [comment]
Sorry for the delayed reply, are you using Flatpak or AppImage?

**Comment by SportingChap** [comment]
> Sorry for the delayed reply, are you using Flatpak or AppImage?

I'm using the AppImage.

**Comment by dangeredwolf** [comment]
ModernDeck has been updated to version 10.0.0, is the issue you were facing still present in the new version?

**Comment by SportingChap** [comment]
Yes, looks like nothing has changed. No icon in the start menu. And even the custom icon I set for the panel pin has been wiped out by the update.
